api/shared/nodenetworkconfigurationpolicy_types.go

```diff
@@ -12,6 +14,15 @@ type NodeNetworkConfigurationPolicySpec struct {
 	// +kubebuilder:validation:XPreserveUnknownFields
 	// The desired configuration of the policy
 	DesiredState State `json:"desiredState,omitempty"`
+
+	// +kubebuilder:validation:XPreserveUnknownFields
+	// When Parallel is set to true, changes are applied to all nodes in parallel
+	// +optional
+	Parallel bool `json:"parallel,omitempty"`
+
+	NodeRunningUpdate string `json:"nodeRunningUpdate,omitempty" optional:"true"`
+
+	NodeUpdateStart metav1.Time `json:"nodeUpdateStarted,omitempty" optional:"true"`
```

Comment:
These two should be in PolicyState, not Spec, no?

Reply:
Yes, moved to PolicyState